java/ql/test/library-tests/frameworks/apache-collections/test.ql

```diff
@@ -9,6 +9,9 @@ class SummaryModelTest extends SummaryModelCsv {
     row =
       [
         //"package;type;overrides;name;signature;ext;inputspec;outputspec;kind",
+        "org.apache.commons.collections4.iterators;IteratorEnumeration;true;IteratorEnumeration;;;Element of Argument[0];Element of Argument[-1];value",
+        "generatedtest;Test;false;getMapKeyFromPut;;;MapKey of Argument[0];ReturnValue;value",
+        "generatedtest;Test;false;getMapValueFromPut;;;MapValue of Argument[0];ReturnValue;value",
```

Comment:
Remove these 3 lines. For the latter 2, I've given implementation suggestions (I believe classes implementing `Put` typically also implement `Map`). For `IteratorEnumeration` I don't quite understand why you've put it here. Shouldn't that just be part of the modelling in `Collections.qll`?

Reply:
`IteratorEnumeration` will be modelled in the next PR (which I'm working on now). But it would be very convenient to have its model now, so that I can make an enumeration with untrusted data in one of its elements. I've put this line here temporarily, and it will be removed when the proper model (+ tests) is added in the next PR.